lib/ansible/modules/network/f5/bigip_command.py

```diff
@@ -78,6 +78,17 @@
         conditional, the interval indicates how to long to wait before
         trying the command again.
     default: 1
+  transport:
+    description:
+      - Configures the transport connection to use when connecting to the
+        remote device. The transport argument supports connectivity to the
+        device over cli (ssh) or rest.
+    required: true
+    choices:
+        - rest
```

Comment:
@caphrim007 is `rest` how you describe the interface?

Reply:
@gundalow yes